src/Components/Shared/src/WebEventData.cs

```diff
@@ -3,57 +3,36 @@
 
 using System;
 using System.Text.Json;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Components.Rendering;
-using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.Web
 {
-    /// <summary>
-    /// Provides mechanisms for dispatching events to components in a <see cref="Renderer"/>.
-    /// </summary>
-    public static class RendererRegistryEventDispatcher
+    internal class WebEventData
```

Comment:
I don't have strong feelings about making this public or not. I agree that it's not a big maintanance burden to make this public if we think its necessary.

Reply:
Let's go with the risk-averse option for now. It's fine being shared source currently and we can change it later if we want.